source/_patterns/components/03-cards/01-card-icon-large.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Card
+---
+## Overview
+
+This is a Bootstrap component, [see complete documentation here](http://v4-alpha.getbootstrap.com/components/card/)
+
+## Usage
+
+| Class | Usage |
+| -- | -- |
+| `.text-xs-center` **Applied to:** `.card` |  **Outcome:** Center-aligns the contents of the card   **Required:** No  |
```

Comment:
same here with text center. It's a utility class. Do you think we should document it anyway even though we should document the same for every element?